### ISSUE TITLE
Removed Tweak Icon stay on home screen

### ISSUE THREAD

**Issue by AztekGod** [checkra1n/BugTracker#62]
**What device + iOS version are you on?**
13.2.2

**What are the steps to reproduce the issue?**
1.Open Cydia, removed tweak from there
2.Repring and icon stay there
3.Try to open and it just crashes 
...

**What do you expect, and what is happening instead?**
Tweak to go away

**Any other info, error logs, screenshots, ...?**
No errors

**Comment by KhafraDev** [comment]
I'm not jailbroken, but just run "uicache" or "uicache -a" in terminal/SSH to update the icon cache. I'm not sure if they use the old or new uicache so try using both.

**Comment by Shadowedstalker** [comment]
Yes just SSH and uicache will remove the issues. Waiting until the phone rebuilds the cache will also if you are unsure of ssh.

**Comment by mmarquez661** [comment]
Same issue....Only way for me to remove the tweak is to SSH from my Mac to phone and use uicache

**Comment by Shadowedstalker** [comment]
You can also download NewTerm 2 via Cydia and rebuild the cache from the phone.

Open NewTerm 2 type uicache press enter.
